### PR TITLE
Add Lucent Sky AVM

### PR DESCRIPTION
**Pull request by sandy-lucentsky**

### CODE REVIEW

File: _data/tools.json [hunk 1]
```diff
@@ -1816,5 +1816,14 @@
...
       "type": "DAST"
-   }  
+   },
+   {
+      "title": "Lucent Sky AVM",
+      "url": "https://www.lucentsky.com/avm",
+      "owner": "Lucent Sky",
+      "license": "Commercial",
+      "platforms": "SaaS or On-Premises",
+      "note": "Automatic finds and fixes application vulnerabilities in source code. Supported .NET, ASP, Android, C and C++, ECMAScript, Go, iOS, Java, PHP, Pytohn, Ruby, and Visual Basic applications.",
```
Comment (kingthorin) [inline _data/tools.json:1826]: Automatically?

Comment (sandy-lucentsky) [reply _data/tools.json:1826]: The tool finds vulnerabilities in source code and generates source code patches to replace vulnerable source code to fix the vulnerabilities.

If automatically is not an ideal word to use in this context, does the above sound better?

Comment (kingthorin) [reply _data/tools.json:1826]: I was suggesting you change “Automatic” ➡️ “Automatically”

Comment (sandy-lucentsky) [reply _data/tools.json]: Must have made the error while copy-n-pasting it 😳. Thanks for catching it!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**